templates/esm/index.html

```diff
@@ -43,6 +43,18 @@ <h4>ITstrategen keeps their applications secure with ESM</h4>
 </section>
 
 <section class="p-strip is-deep is-bordered" id="get-esm">
+  <div class="row">
+    <div class="col-8">
+      <h2>Free for personal use</h2>
+      <p>Canonical provides Ubuntu Advantage Essential subscriptions, which include ESM, free of charge for individuals on up to 5 machines. For our community of <a href="https://wiki.ubuntu.com/Membership">Ubuntu members</a> we will gladly increase that to 50 machines. Your personal subscription will also cover <a href="/livepatch">Livepatch</a>, FIPS and CIS hardening tools.</p>
+      <a href="/advantage#esm" class="p-button--positive u-no-margin--bottom">Get ESM now</a>
```

Comment:
This section is about the free subscription, but the link goes into the table describing paid subscriptions.

I suggest dropping this separate “Free for personal use” section. “And it’s free for personal use” above is fine.